### ISSUE TITLE
enhancement: Alert dev team if a failed search index update goes over the max retry count

### ISSUE THREAD

**Issue by hholb** [Garden-AI/garden-backend#132]
In #126 we added logic to track and retry failed updates to the search index. It would be nice to have something in place that will alert the dev team to failed updates that have been around long enough to go over the `MAX_RETRY_COUNT`.

Possible solution:
- Setup a bot that can send a message to the garden-errors slack channel

**Comment by hholb** [comment]
No longer as issue after #155
